docs/src/main/sphinx/installation/query-resiliency.rst

```diff
@@ -0,0 +1,120 @@
+===================================
+Improve query processing resilience
+===================================
+
+You can configure Trino to be more resilient against failures during query
+processing by enabling fault-tolerant execution. This allows Trino to handle
+larger queries such as batch operations without worker node interruptions
+causing the query to fail.
+
+When configured, the Trino cluster buffers data used by workers
+during query processing. If a worker node fails for any reason, such as a
+network outage or running out of available resources, the coordinator issues the
+data to an available worker that can pick up query processing, allowing query
+processing to continue.
```

Comment:
```suggestion
When configured, the Trino cluster buffers data used by workers
during query processing. If a processing on a worker node fails for any reason, such as a
network outage or running out of available resources, the coordinator reschedules the processing of failed piece of work using buffered data on the same on another worker, possibly adjusting resouce allocation limits. This allows query  processing to continue.
```

Reply:
Edited a bit further for readability, please call out if the rewrite is incorrect.